src/modules/blockchainApplication/explore/store/action.js

```diff
@@ -0,0 +1,16 @@
+import actionTypes from './actionTypes';
+
+/**
+ * Trigger this action to set blockchain application pin
+ *
+ * @returns {Object} - Action object
+ */
+export const pinApplication = (chainId) => ({
+  type: actionTypes.setApplicationPin,
+  chainId,
```

Comment:
@eniolam1000752 For consistency with the rest of the actions, data should be the key for the Redux action, same way type is always a key